iceberg/azure/LICENSE

```diff
@@ -0,0 +1,337 @@
+
```

Comment:
Why is this file needed?

Reply:
+

Reply:
Levi said to include because we are copying this code